src/App.jsx

```diff
@@ -1,7 +1,28 @@
-import data from "./data.json";
+import data from './data.json'
+import moreData from './stretch-goal.json'
+import { Header } from './assets/Components/Header/Header.jsx'
+import { Sidebar } from './assets/Components/Sidebar/Sidebar.jsx'
 
-console.log(data);
+import { Section } from './assets/Components/Section/Section.jsx'
+import './App.css'
+
+console.log(data)
+console.log(moreData)
 
 export const App = () => {
-  return <div>Find me in src/app.jsx!</div>;
-};
+  return (
+    <div className="App">
+      <Header />
+
+      <div id="albumContainer" className="album-container">
+        <Section name="Single" filter="single" data={data} />
+        <Section name="Album" filter="album" data={data} />
+      </div>
+      <div className="sideBar">
+        {moreData.playlists.items.map((playlists, index) => (
+          <Sidebar key={index} playlists={playlists} />
+        ))}
```

Comment:
Very nice that you accomplished the stretch goal! But consider thinking about the display on mobile, as mentioned